app/src/main/java/com/m2049r/xmrwallet/LoginActivity.java

```diff
@@ -959,6 +965,45 @@ void copyFile(File src, File dst) throws IOException {
         }
     }
 
+    public void onChangeLocale() {
+        final ArrayList<Locale> translatedLocales = LocaleHelper.getAvailableLocales(LoginActivity.this);
+        String[] localeDisplayName = new String[1 + translatedLocales.size()];
+
+        Collections.sort(translatedLocales, new Comparator<Locale>() {
+            @Override
+            public int compare(Locale locale1, Locale locale2) {
+                return locale1.getDisplayName(locale1).compareTo(locale2.getDisplayName(locale2));
+            }
+        });
+
+        localeDisplayName[0] = getString(R.string.language_system_default);
+        for (int i = 1; i < localeDisplayName.length; ++i) {
+            Locale locale = translatedLocales.get(i - 1);
+            localeDisplayName[i] = locale.getDisplayName(locale);
```

Comment:
this results in some languages being capitalised and others not - the list doesn't look good.

Reply:
I can capitalize first letter of display name, such as Language setting in System Settings.
How about this idea?